### PR TITLE
updated jenkins CI sample snippet

### PR DESCRIPTION
**Pull request by holdenmcg**
Updated the Jenkins snippet in the Sample CI Configurations section to no longer use Docker and switch it to using semgrep ci.

image

# Thanks for improving Semgrep Docs 😀

### Please ensure

- [ ] A subject matter expert (SME) reviews the content
- [ ] A technical writer reviews the content or PR
- [ ] This change has no security implications or else you have pinged the security team
- [ ] Redirects are added if the PR changes page URLs

### CODE REVIEW

File: docs/semgrep-ci/sample-ci-configs.md [hunk 1]
```diff
@@ -161,55 +161,27 @@ Use webhooks and the below snippet to integrate with GitHub.
...
-  }
-
-environment {
-    SEMGREP_RULES = "p/security-audit p/secrets" // more at semgrep.dev/explore
-    SEMGREP_BASELINE_REF = "origin/${env.CHANGE_TARGET}"
-
-    // == Optional settings in the `environment {}` block
+  agent any
+    environment {
+      SEMGREP_RULES = "p/security-audit p/secrets" // more at semgrep.dev/explore
```
Comment (ievans) [inline docs/semgrep-ci/sample-ci-configs.md:166]: can we change to auto?

Comment (holdenmcg) [reply docs/semgrep-ci/sample-ci-configs.md:166]: Sure thing; as in, removing the "SEMGREP_RULES" variable and adding "--config auto . " to the command?

Comment (brendongo) [reply docs/semgrep-ci/sample-ci-configs.md:166]: The latter would be great @holdenmcg

Comment (holdenmcg) [reply docs/semgrep-ci/sample-ci-configs.md]: Commented out the 'environment' block, setting it to --config auto by default.

File: docs/semgrep-ci/sample-ci-configs.md [hunk 2]
```diff
@@ -161,55 +161,27 @@ Use webhooks and the below snippet to integrate with GitHub.
...
-    stage('Semgrep_agent') {
-      when {
-        // Scan changed files in PRs, block on new issues only (existing issues ignored)
-        expression { env.CHANGE_ID && env.BRANCH_NAME.startsWith("PR-") }
-      }
-      steps {
-        sh 'git fetch origin ${SEMGREP_BASELINE_REF#origin/} && semgrep-agent'
+  agent any
+    // environment {
+      // SEMGREP_RULES = "p/security-audit p/secrets" // more at semgrep.dev/explore
```
Comment (brendongo) [inline docs/semgrep-ci/sample-ci-configs.md:166]:
```suggestion
```

Comment (brendongo) [reply docs/semgrep-ci/sample-ci-configs.md:166]: I think we can just remove this and people should use --config below

**Review by brendongo — APPROVED** [review]

**Pull request merged**